src/SUMMARY.md

```diff
@@ -1,3 +1,7 @@
 # Summary
 
 [Notice](./notice.md)
+
+# Operations
+
+- [Managing Secrets](operations/secrets.md)
```

Comment:
```suggestion
- [Managing Secrets](operations/secrets.md)
```

You can't have other heading sections in the SUMMARY.md file, it's just bullet points that can be nested. At some point we can create a general operations intro chapter, at which point the SUMMARY.md will look like this:
```markdown
- [Operations](operations/intro.md)
	- [Managing Secrets](operations/secrets.md)
```